api/test/fixtures/users.yml

```diff
@@ -18,6 +18,7 @@
 #  remember_created_at    :datetime
 #  reset_password_sent_at :datetime
 #  reset_password_token   :string
+#  username               :string
```

Comment:
Ty running `cd api && docker-compose run server rails db:migrate` and see if that fixes this bc this username field should not be here